test/tools/libtesteth/TestOutputHelper.cpp

```diff
@@ -32,7 +32,7 @@ using namespace boost;
 
 Timer TestOutputHelper::m_timer;
 size_t TestOutputHelper::m_currTest = 0;
-size_t TestOutputHelper::m_maxTests = 0;
+size_t TestOutputHelper::m_maxTests = 1;
```

Comment:
there was a division by 0 if TestOutputHelper not being initialized. with init function